packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+
+import { Logger } from 'aws-amplify';
+import { changePassword, translate } from '@aws-amplify/ui';
+
+import { useAuth } from '../../../internal';
+import { View, Flex } from '../../../primitives';
+import {
+  DefaultCurrentPassword,
+  DefaultError,
+  DefaultNewPassword,
+  DefaultSubmitButton,
+} from './defaultComponents';
+import { ChangePasswordProps } from './types';
+import { FormValues } from '../types';
+
+const logger = new Logger('ChangePassword');
+
+const ChangePassword: React.ComponentType<ChangePasswordProps> = (props) => {
+  const { onSuccess, onError } = props;
```

Comment:
Nit: destructure in function declaration

Reply:
Done: https://github.com/aws-amplify/amplify-ui/blob/97936eec8e4fce7511c6c3c5deee9aed98b9984b/packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx#L19-L22